packages/functional-tests/playwright.config.ts

```diff
@@ -1,9 +1,12 @@
-import { PlaywrightTestConfig, Project } from '@playwright/test';
+import { PlaywrightTestConfig } from '@playwright/test';
+import type { Project } from '@playwright/test';
```

Comment:
Is this used somewhere?

Reply:
Type info on L53 below:
https://github.com/mozilla/fxa/blob/16b5d5781f3f61f77965d66be856e8de953987a0/packages/functional-tests/playwright.config.ts#L53